modules/roll/index.js

```diff
@@ -1,6 +1,6 @@
 const Roll = require('roll');
 
-const regex = /^(\d*)d(\d+|\%)(([\+\-\/\*b])(\d+))?$/;
+const regex = /^(\d*)d(\d+|%)(([+\-/*b])(\d+))?$/;
```

Comment:
We should verify that this works the way we want it to.

Reply:
I tossed in a commit with unit tests for roll and the tests still pass so I'm pretty sure it's okay :)